src/sdk-reference/js/6/collection/delete-specifications/index.md

```diff
@@ -7,39 +7,33 @@ description: Delete validation specifications for a collection
 
 # deleteSpecifications
 
-Delete the validation specifications associated with the collection.  
+Delete the validation specifications associated with the `<collection>`.
 
-## Signature
+## Arguments
 
 ```javascript
-/**
- * @param {string} index
- * @param {string} collection
- * @param {object} [options]
- * @returns {Promise.<boolean>}
- */
-deleteSpecifications(index, collection, options = {})
+deleteSpecifications (index, collection, [options])
 ```
 
-## Arguments
+<br/>
 
-| Arguments    | Type    | Description | Required
-|--------------|---------|-------------|----------
-| ``index`` | String | Index name    | yes  |
-| ``collection`` | String | Collection name    | yes  |
-| ``options`` | Object | Query options    | no  |
+| Arguments    | Type    | Description |
+|--------------|---------|-------------|
+| ``index`` | <pre>string</pre> | Index name    |
+| ``collection`` | <pre>string</pre> | Collection name    |
+| ``options`` | <pre>object</pre> | Query options    |
 
 ### **options**
 
 Additional query options
 
-| Property   | Type    | Description                       | Default |
-| ---------- | ------- | --------------------------------- | ------- |
-| `queuable` | boolean | Make this request queuable or not | `true`  |
+| Property     | Type<br/>(default)    | Description   |
+| -------------- | --------- | ------------- |
+|  `queuable`  |  <pre>boolean</pre> <br/>(`true`) |  Make this request queuable or not  |
 
 ## Resolve
 
-True if success
+True if success.
```

Comment:
I think this method never return to false. The method return true because the API return true
https://docs.kuzzle.io/api-documentation/controller-collection/delete-specifications/